packages/core/src/modules/proofs/protocol/v2/errors/V2PresentationProblemReportReason.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Presentation error code in RFC 0037.
+ *
+ * @see https://github.com/hyperledger/aries-rfcs/blob/main/features/0037-present-proof/README.md
+ */
+export enum V2PresentationProblemReportReason {
```

Comment:
Isn't this the same as v1? Can't we make them extend from some shared `PresentationProblemReportReason`?